Lib/tkinter/test/test_tkinter/test_colorchooser.py

```diff
@@ -0,0 +1,42 @@
+import unittest
+import tkinter
+from tkinter import colorchooser
+from test.support import requires, run_unittest, gc_collect
+from tkinter.test.support import AbstractTkTest
+
+requires('gui')
+
+class ChooserTest(AbstractTkTest, unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        AbstractTkTest.setUpClass.__func__(cls)
+        cls.cc = colorchooser.Chooser(initialcolor='red')
+
+    def test_fixoptions(self):
+        cc = self.cc
+        cc._fixoptions()
+        self.assertEqual(cc.options['initialcolor'], 'red')
+
+        cc.options['initialcolor'] = '#ffff00000000'
```

Comment:
Please use something more interesting than the sequence of `f`'s and `0`'s. And in other tests too.